1.9/api/versioning.md

```diff
@@ -0,0 +1,31 @@
+---
+post_title: Versioning
+menu_order: 2
+---
+
+The DC/OS API is backed by many loosely coupled components, some of which are standalone projects and some of which were designed exclusively for DC/OS. Because of this, several types of versioning mechanisms are supported: component, route, and resource versioning.
+
+To learn how to formulate a specific API call, see the component API reference documentation for that route.
+
+## Component Versioning
+
+Components with their own open source communities, like Mesos, Marathon, and Mesos DNS, have routes that are based on the well known component name. These routes delegate versioning to the backend component service.
+
+For example, the [Marathon component](/docs/1.9/overview/architecture/components/#marathon) serves the [Marathon API](/docs/1.9/deploying-services/marathon-api/) under the route `/service/marathon` and one of its resource paths is `/v2/apps`, so the full path to that resource is `/service/marathon/v2/apps`.
+
+## Route Versioning
+
+Components that have been specifically designed exclusively for DC/OS tend to follow another versioning pattern, where the name of the component takes a back seat to the name of the feature set. These routes often include a version to make it easier to support renaming or replacing components over time.
```

Comment:
- Think you should pick either `exclusively designed for DC/OS` or `specifically designed for DC/OS`, but not both. 
- `tend to follow` - don't they *always* follow another versioning scheme? if not, then suggest the word `...generally follow another versioning...` instead.
-